eLearning-courseware/eLearning.jsonld

```diff
@@ -0,0 +1,1271 @@
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/matching",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Matching"
+        },
+        "definition": {
+          "en": "Learners must match labeled items to the correct bucket."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/multipleChoice",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Multiple choice"
+        },
+        "definition": {
+          "en": "A single question with multiple answers for the learner to choose from."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/sortable",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Sortable"
+        },
+        "definition": {
+          "en": "A single question with multiple answers that must be sorted in the correct order."
+        }
+      },
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/activities/videoBranching",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Video branching"
+        },
+        "definition": {
+          "en": "The learner watches a video and is then asked a question. Depending on how they answer, they are taken to another video or another question."
+        }
+      }
+    ],
+    "templates": [
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/coursePassed",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "prefLabel": {
+          "en": "Course passed"
+        },
+        "definition": {
+          "en": "Learner has passed the course."
+        },
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.name",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.result.success",
+            "any": [true]
+          },
+          {
+            "location": "$.result.completion",
+            "any": [true]
+          },
+          {
+            "location": "$.result.score",
+            "presence": "included"
+          },
+          {
+            "location": "$.verb.id",
```

Comment:
This should be turned into the `verb` property -- right now, this template applies to all statements (using the profile version in category), but changing this to `verb` will change this template to only apply to statements using the profile version in category _and_ using the same verb as in the verb property.

Basically, the "determining properties" such as verb decide which statement templates apply -- if you want to apply to all statements from the profile, have no determining properties, and if you only want to apply to some, put in some mix of verb, an objectActivityType, a contextGroupingActivityType, etc.

Then, once a statement template is determined to apply for a statement, all the rules need to be followed for the statement to follow the profile.

Reply:
This makes sense. I overlooked the `verb` property but now I understand it. I changed all the templates to use the `verb` property instead of a rule.